region/maridia/inner-pink/Aqueduct.json

```diff
@@ -887,6 +988,79 @@
         {"shinespark": {"frames": 40, "excessFrames": 7}}
       ]
     },
+    {
+      "link": [2, 7],
+      "name": "Temporary Blue Gravity Jump",
+      "requires": [
+        "h_canShineChargeMaxRunway",
+        "canGravityJump",
+        "canChainTemporaryBlue"
```

Comment:
HiJump should be enough to only use a regular tempblue grav jump

Reply:
Yeah, good find, it does work.